tests/ignite/metrics/test_root_mean_squared_error.py

```diff
@@ -17,24 +18,64 @@ def test_zero_div():
 
 
 def test_compute():
-    rmse = RootMeanSquaredError()
 
-    y_pred = torch.Tensor([[2.0], [-2.0]])
-    y = torch.zeros(2)
-    rmse.update((y_pred, y))
-    assert isinstance(rmse.compute(), float)
-    assert rmse.compute() == 2.0
+    rmse = RootMeanSquaredError()
 
-    rmse.reset()
-    y_pred = torch.Tensor([[3.0], [-3.0]])
-    y = torch.zeros(2)
-    rmse.update((y_pred, y))
-    assert isinstance(rmse.compute(), float)
-    assert rmse.compute() == 3.0
+    def _test(y_pred, y, batch_size):
+        rmse.reset()
+        rmse.update((y_pred, y))
+
+        np_y = y.numpy().ravel()
+        np_y_pred = y_pred.numpy().ravel()
+
+        if batch_size > 1:
+            n_iters = y.shape[0] // batch_size + 1
+            for i in range(n_iters):
+                idx = i * batch_size
+                rmse.update((y_pred[idx : idx + batch_size], y[idx : idx + batch_size]))
+
+        np_res = np.sqrt(np.power((np_y - np_y_pred), 2.0).sum() / np_y.shape[0])
+        res = rmse.compute()
+
+        assert isinstance(res, float)
+        assert pytest.approx(res) == np_res
+
+    def get_test_cases():
+
+        test_cases = [
+            (
+                torch.randint(0, 10, size=(50,), dtype=torch.float),
+                torch.randint(0, 10, size=(50,), dtype=torch.float),
+                1,
+            ),
+            (
+                torch.randint(-10, 10, size=(50, 1), dtype=torch.float),
+                torch.randint(-10, 10, size=(50, 1), dtype=torch.float),
+                1,
+            ),
+            # updated batches
+            (
+                torch.randint(0, 10, size=(50,), dtype=torch.float),
+                torch.randint(0, 10, size=(50,), dtype=torch.float),
+                16,
```

Comment:
Same here for `torch.rand()` and rescale it to [0, 10] interval,
```suggestion
                torch.randint(0, 10, size=(50,), dtype=torch.float),
                torch.randint(0, 10, size=(50,), dtype=torch.float),
                16
```